building/trusted_applications.rst

```diff
@@ -299,3 +305,90 @@ sequence:
             return TEE_ERROR_NOT_SUPPORTED;
         }
     }
+
+.. _TASign:
+
+Signing of TAs
+**************
+
+All Trusted Applications are signed and verified by :ref:`optee_os` upon loading. Within the :ref:`optee_os` source is a directory ``keys``. The public parts of the keys in this directory will be compiled into the :ref:`optee_os` binary and the signature of each TA will be verified against this key(s) upon loading.
```

Comment:
Comma after `signed`? (TAs are obviously not signed "upon loading" so one should not read "signed and verified" together).
Why `keys` and `key(s)`? (plural). I'd rather be more precise: "The public part of `keys/default_ta.pem` will be [...] against this key [...]".